vector/src/main/java/im/vector/app/features/discovery/DiscoverySettingsController.kt

```diff
@@ -71,6 +70,7 @@ class DiscoverySettingsController @Inject constructor(
                     buildMsisdnSection(data.phoneNumbersList)
                 }
             }
+            else       -> Unit
```

Comment:
It would be better to explicitly define `Uninitialized -> Unit` than use `else -> Unit`. Reason being if a new subclass of `Async`, the compiler would catch that everywhere we use it.

In this case it's unlikely that airbnb will add one, but in general it's good practice